documentation/user_docs/docs/conf.py

```diff
@@ -0,0 +1,58 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Horace'
+copyright = '2020, Alex Buts, Toby Perring, Nick Battam, Harry Saunders, Duc Le, Chris Marooney'
+author = 'Alex Buts, Toby Perring, Nick Battam, Harry Saunders, Duc Le, Chris Marooney'
```

Comment:
Should this list be alphabetical or start with primary contacts names?

Russell Ewings should be included as he has made significant contributions to the project in the past.
There may be others authors, again check with Toby.

Reply:
From `horace_on` 
![image](https://user-images.githubusercontent.com/48721877/100071848-6c4b0e00-2e33-11eb-9536-b200654c60ba.png)

Reply:
That's the reference paper not the copyright holders or authors. I've dropped a message to Toby, so we'll find out at some point and I'll update it.

Reply:
No, I don't have a list of Horace contributors. or the copyright owner. There are a number of interesting questions here. 
- I think we should maintain a formal list of contributors to the parts of PACE - don't know where is best. 
- The Horace list at present should be the super set of the current HORACE team and the paper. That means (off the top of my head) Nick, Harry, Jacob, Chris, Greg and I think Becky.
- There is a Euphonic list, Brille list, PACE list too. These lists are more than those who've actually written code - any intellectual contribution should count.
- Order should be alphabetical
- Copyright is a good question. I'll ask what we should do here (if there is an STFC policy at all of course)